common/src/main/java/com/wynntils/screens/itemsharing/SavedItemsScreen.java

```diff
@@ -0,0 +1,528 @@
+                    }
+                },
+                List.of(
+                        Component.translatable("screens.wynntils.savedItems.changeCategoryTooltip1"),
+                        Component.translatable("screens.wynntils.savedItems.changeCategoryTooltip2")),
+                Texture.VAULT_CONFIRM));
+        // endregion
+
+        // region Category navigation buttons
+        this.addRenderableWidget(new SavedCategoryButton(this.leftPos + 19, this.topPos + 9, this, false));
+
+        this.addRenderableWidget(new SavedCategoryButton(this.leftPos + 139, this.topPos + 9, this, true));
+        // endregion
+
+        // region Info button
+        this.addRenderableWidget(new InfoButton(
+                15,
+                15,
+                Component.literal("")
+                        .append(Component.translatable("screens.wynntils.savedItems.help")
+                                .withStyle(ChatFormatting.UNDERLINE))
+                        .append(Component.literal("\n"))
+                        .append(Component.translatable("screens.wynntils.savedItems.help1")
+                                .withStyle(ChatFormatting.GRAY))
+                        .append(Component.literal("\n"))
+                        .append(Component.translatable("screens.wynntils.savedItems.help2")
+                                .withStyle(ChatFormatting.GRAY))
+                        .append(Component.literal("\n"))
+                        .append(Component.translatable("screens.wynntils.savedItems.help3")
+                                .withStyle(ChatFormatting.GRAY))));
+        // endregion
+
+        // Re add input box if screen resized during input
+        if (addingCategory) {
+            addCategoryInput();
+        }
+
+        populateItems();
+    }
+
+    @Override
+    protected void renderBg(GuiGraphics guiGraphics, float partialTick, int mouseX, int mouseY) {
+        PoseStack poseStack = guiGraphics.pose();
+
+        RenderUtils.drawTexturedRect(poseStack, Texture.VAULT, this.leftPos, this.topPos);
+    }
+
+    @Override
+    protected void renderLabels(GuiGraphics guiGraphics, int mouseX, int mouseY) {
+        // We don't want a label
+    }
+
+    @Override
+    protected void slotClicked(Slot slot, int slotId, int mouseButton, ClickType type) {
+        if (slot == null) return;
+        if (slot.getItem() == ItemStack.EMPTY) return;
+
+        // Left click goes to sharing menu
+        // Left+Shift deletes the item from storage
+        // Right click selects it for category change
+        if (mouseButton == 0) {
+            if (KeyboardUtils.isShiftDown()) {
+                deleteItem(encodedItems.get(slot.index));
+            } else {
+                Optional<WynnItem> wynnItemOpt = Models.Item.getWynnItem(slot.getItem());
+
+                if (wynnItemOpt.isPresent()) {
+                    McUtils.mc().setScreen(ItemSharingScreen.create(wynnItemOpt.get(), slot.getItem(), true));
```

Comment:
Left click action probably not needed, can just use F3 on item and ChatItemFeature can handle it

Reply:
Hm. I think I probably like the left click option as well, if we have no other use for left click.